lib/sinon/stub-non-function-property.js

```diff
@@ -3,15 +3,13 @@
 var valueToString = require("./util/core/value-to-string");
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
-function stubNonFunctionProperty(object, property, value) {
+function stubNonFunctionProperty(object, property) {
     var original = object[property];
 
     if (!hasOwnProperty.call(object, property)) {
         throw new TypeError("Cannot stub non-existent own property " + valueToString(property));
     }
 
-    object[property] = value;
```

Comment:
Looks like most of the property tests were removed. I see negative tests, but no positive tests.